README.md

```diff
@@ -1,31 +1,207 @@
 
-## Backend
-The Neon Backend is a collection of services that are provided to client applications.
-Some backend services connect via a RabbitMQ server, while others are served via HTTP.
+After you have installed Diana, use `diana --help` to get detailed help.
 
-### Prerequisites
-- `python3` is required to install `neon-diana-utils`
-- `kubectl` and `helm` are needed to apply the generated configurations
+For more information about using NeonAI, see [the NeonAI documentation site](https://neongeckocom.github.io/neon-docs/).
 
-### Configure Backend with `diana`
-A Diana backend can be configured automatically with `diana configure-backend`. A standard example is included here, but 
-a description of config options is available via: `diana configure-backend --help`.
+## Prerequisites
 
-```shell
+* [Ubuntu](https://ubuntu.com/) 20.04 or later. 
+  * Diana will most likely work on other flavors of Linux, but we have not yet verified this.
+* [Python](https://www.python.org/getit/) 3.8 or later.
+* [Pip](https://pypi.org/project/pip/) Python package installer.
+* [Kubectl](https://kubernetes.io/docs/reference/kubectl/) Kubernetes command-line tool.
+* [Helm](https://helm.sh/) package manager for Kubernetes.
+* A [Kubernetes](https://kubernetes.io/) installation.
+  * The following instructions assume a local installation using [Microk8s](https://microk8s.io/) version 1.26/stable or later.
+  * You can likely deploy Diana on a Kubernetes cluster in the cloud, but we have not yet verified this.
+* [Neon Skill Utilities](https://github.com/NeonGeckoCom/neon-utils) 1.0 or later.
+  * Use `pip show neon-utils` to check the version installed on your system.
+  * Use `pip install neon-utils --upgrade` to upgrade Neon Skill Utilities to the most recent version.
+
+
+## Use a Python virtual environment
+
+We recommend you use a [Python virtual environment](https://docs.python.org/3/library/venv.html) for installing and testing Diana. In addition to the usual benefits, using a virtual environment makes it easier to ensure you are using the correct versions of Python and Diana.
+
+1. Create a Python virtual environment:
+
+```
+python3.10 -m venv venv
+```
+
+2. Open a new terminal window. Activate the Python virtual environment in this new window:
+
+```
+. venv/bin/activate
+```
+
+Using this new window, proceed with the instructions.
+
+## Install Diana
+
+3. Use Pip to install Diana:
+
+```
+pip install --pre neon-diana-utils
+```
+
+This command installs the newest pre-release version, which is described in this tutorial. 
+
+**Warning:** You can use `pip install neon-diana-utils` to install the current stable version. However, that version does not include Helm chart support. For information on installing and running the current stable version, see [the archived documentation in the README file](https://github.com/NeonGeckoCom/neon-diana-utils/blob/dev/README.md).
```

Comment:
Maybe update this note to indicate that Helm support is (will be?) added in version `0.1.0` so we don't have to edit it before release